### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torchvision'

### DISCUSSION THREAD

**Discussion by anadikulkarni** [mrdbourke/pytorch-deep-learning#1018] [Q&A]
Video number: 71

This happens when I try to use `helper_functions.py`. My code doesn't run because there is an error in the .py file. I have torchvision installed on my local computer. I am not running my code in a virtual environment.

**Comment by LaurencePy** [comment]
ensure you have used pip package manager to pip install. Have pip updated. Then import in the code. 
If nothing works then I suggest you make a virtual environment and reinstall.
